packages/flutter/lib/src/services/system_channels.dart

```diff
@@ -54,4 +54,10 @@ class SystemChannels {
       const JSONMessageCodec(),
   );
 
+  /// A JSON [BasicMessageChannel] for accessibility events.
```

Comment:
This needs much more documentation. What are all the methods? What are their semantics? etc.